### PR TITLE
The 1.16 PR

### LINKED ISSUE #865: Protect Respawn Anchors in 1.16
1.16 adds [Respawn Anchors](https://minecraft.gamepedia.com/Respawn_Anchor) which act like a bed in the overworld to set your respawn location for later use with `/home` etc.

They have to be "charged" with glowstone, and each use consumes a charge.

At present, people untrusted in the claim can interact with them and thus consume the owner's charges, leaving the owner unable to /home.

They should probably be protected from interactions unless you have ContainerTrust.

Thanks to @IndigoRoger for reporting.

### PR DESCRIPTION
**Pull request by isabelcoolaf**
This probably isn't perfect so go easy on me but lmk.
It compiles and in my testing it works.

Also this fixes #865

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -71,9 +71,9 @@
     <dependencies>
         <!--Bukkit API-->
         <dependency>
-                <groupId>org.bukkit</groupId>
-                <artifactId>bukkit</artifactId>
-                <version>1.15.2-R0.1-SNAPSHOT</version>
+                <groupId>org.spigotmc</groupId>
+                <artifactId>spigot-api</artifactId>
```
Comment (isabelcoolaf) [inline pom.xml:75]: This is changed due to the fact SpigotMC no longer publishes Bukkit jar files and there isn't any other Bukkit Maven repo.

Source: https://www.spigotmc.org/threads/spigot-bungeecord-1-16-1.447405/

**Comment by Jikoo** [comment]
Surprisingly small, is that all there was to 1.16? I guess the mounts are covered by existing event handling. Looks clean :+1:

**Comment by isabelcoolaf** [comment]
1.16 didn't add much in terms of plugin dev /shrug

**Review by Jikoo — CHANGES_REQUESTED** [review]
Needs api-version bump

**Comment by isabelcoolaf** [comment]
Woops, good point! Fixed.

**Review by Jikoo — APPROVED** [review]

**Comment by RoboMWM** [comment]
small diff so that's good, else I prolly would've gone off the whole "one PR per bug fix!" mini rant

File: pom.xml (hunk 1 above)
Comment (RoboMWM) [reply pom.xml:75]: Interesting.

With that, I'm sorta tempted to just set it to Paper at that rate then.

**Review by RoboMWM — APPROVED** [review]
kewl, thanks.

**Comment by bigpresh** [comment]
Just wanted to add my thanks as well.

I've built GP with these changes and tested on my test server, and all seems to be fine, so I've just upgraded our live server to 1.16.1 with that GP build.

**Pull request merged**